src/AppBundle/Charts/Charts.php

```diff
@@ -0,0 +1,132 @@
+<?php
```

Comment:
Is it a service? If yes what do you think to put it in a "Services" folder instead?